src/containers/Configure/Configure.test.js

```diff
@@ -1,3 +1,202 @@
+
+    fireEvent.change(sizeInput, { target: { value: 4 } })
+    fireEvent.change(moduleTypeInput, { target: { value: 0 } })
+    // fireEvent.change(arrayTypeInput, { target: { value: 0 } })
+    fireEvent.change(systemLossesInput, { target: { value: 14 } })
+    fireEvent.change(tiltInput, { target: { value: 20 } })
+    fireEvent.change(azimuthInput, { target: { value: 180 } })
+    fireEvent.click(submitButton)
+
+    const errMessage = getByText('Please select an array type')
+
+    expect(errMessage).toBeInTheDocument()
+  })
+
+  it('should give an error message upon form submission if system losses field is out of range', () => {
+    const { getByText, getByLabelText, debug } = configureTestWrapper()
+
+    const sizeInput = getByLabelText('System Size (kW) (1-10):')
+    const moduleTypeInput = getByLabelText('Module Type:')
+    const arrayTypeInput = getByLabelText('Array Type:')
+    const systemLossesInput = getByLabelText('System Losses (0-20%):')
+    const tiltInput = getByLabelText('Tilt (0-45 degrees):')
+    const azimuthInput = getByLabelText('Azimuth (0-359 degrees):')
+    const submitButton = getByText('Submit')
+
+    fireEvent.change(sizeInput, { target: { value: 4 } })
+    fireEvent.change(moduleTypeInput, { target: { value: 0 } })
+    fireEvent.change(arrayTypeInput, { target: { value: 0 } })
+    fireEvent.change(systemLossesInput, { target: { value: 100 } })
+    fireEvent.change(tiltInput, { target: { value: 20 } })
+    fireEvent.change(azimuthInput, { target: { value: 180 } })
+    fireEvent.click(submitButton)
+
+    const errMessage = getByText('Please reverify inputs to be in acceptable range')
+
+    expect(errMessage).toBeInTheDocument()
+  })
+
+  it('should give an error message upon form submission if tilt field is out of range', () => {
+    const { getByText, getByLabelText, debug } = configureTestWrapper()
+
+    const sizeInput = getByLabelText('System Size (kW) (1-10):')
+    const moduleTypeInput = getByLabelText('Module Type:')
+    const arrayTypeInput = getByLabelText('Array Type:')
+    const systemLossesInput = getByLabelText('System Losses (0-20%):')
+    const tiltInput = getByLabelText('Tilt (0-45 degrees):')
+    const azimuthInput = getByLabelText('Azimuth (0-359 degrees):')
+    const submitButton = getByText('Submit')
+
+    fireEvent.change(sizeInput, { target: { value: 4 } })
+    fireEvent.change(moduleTypeInput, { target: { value: 0 } })
+    fireEvent.change(arrayTypeInput, { target: { value: 0 } })
+    fireEvent.change(systemLossesInput, { target: { value: 14 } })
+    fireEvent.change(tiltInput, { target: { value: 100 } })
+    fireEvent.change(azimuthInput, { target: { value: 180 } })
+    fireEvent.click(submitButton)
+
+    const errMessage = getByText('Please reverify inputs to be in acceptable range')
+
+    expect(errMessage).toBeInTheDocument()
+  })
+
+  it.skip('should give an error message upon form submission if azimuth field is out of range', () => {
```

Comment:
Great job testing all the different sad paths for this component.  Our testing is shaping up to look very thorough!